[{"ContainerToolsException.Message":"Visual Studio Container Tools requires Docker Desktop.  To get it, see https://go.microsoft.com/fwlink/?linkid=847268  For more info, please see: https://aka.ms/DockerToolsTroubleshooting","ContainerToolsException.ErrorCode":"","ContainerToolsException.StackTrace":"","ContainerToolsException.HResult":"-2146233088","ContainerToolsException.InnerException":"Microsoft.VisualStudio.Containers.Tools.Common.ContainerToolsException: Unable to run 'docker-compose'. Verify that Docker Desktop is installed and running locally. For troubleshooting, please refer to https://aka.ms/DockerToolsTroubleshooting. ---> System.ComponentModel.Win32Exception: El sistema no puede encontrar el archivo especificado\r\n   en System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)\r\n   en System.Diagnostics.Process.Start()\r\n   en Microsoft.Docker.Utilities.CommandLineClient.<>c__DisplayClass0_0.<ExecuteAsync>b__0() en D:\\a\\_work\\1\\s\\src\\Microsoft.Docker\\Utilities\\CommandLineClient.cs:línea 172\r\n   --- Fin del seguimiento de la pila de la excepción interna ---\r\n   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   en Microsoft.Docker.DockerComposeClient.<ExecuteAsync>d__13.MoveNext() en D:\\a\\_work\\1\\s\\src\\Microsoft.Docker\\DockerComposeClient.cs:línea 253\r\n--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---\r\n   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   en Microsoft.Docker.DockerComposeClient.<DownAsync>d__8.MoveNext() en D:\\a\\_work\\1\\s\\src\\Microsoft.Docker\\DockerComposeClient.cs:línea 104\r\n--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---\r\n   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   en Microsoft.Docker.DockerWorkspace.<ComposeDownContainersAsync>d__32.MoveNext() en D:\\a\\_work\\1\\s\\src\\Microsoft.Docker\\DockerWorkspace.cs:línea 424\r\n--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---\r\n   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   en Microsoft.Docker.BuildTasks.DockerBuildTask.<ExecuteAsync>d__34.MoveNext() en D:\\a\\_work\\1\\s\\src\\Microsoft.Docker.BuildTasks\\DockerBuildTask.cs:línea 82\r\nCódigo de error: DT1001\r\nGravedad: Error\r\nLínea de comandos: docker-compose  -f \"C:\\Users\\corsair gaming\\source\\repos\\Web_Api_Pedidos\\docker-compose.yml\" -f \"C:\\Users\\corsair gaming\\source\\repos\\Web_Api_Pedidos\\docker-compose.override.yml\" -p dockercompose10914777438481987540 --ansi never kill\r\n"}]